src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -697,15 +697,22 @@ case object CREATE extends OpCode(0xf0, 3, 1, _.G_create) {
 
       val codeDepositGas = state.config.calcCodeDepositCost(result.returnData)
       val gasUsed = startGas - result.gasRemaining + codeDepositGas
+      val enoughGasForDeposit = gasUsed <= availableGas
 
       if (result.error.isDefined) {
         val stack2 = stack1.push(UInt256.Zero)
         state.withStack(stack2).spendGas(startGas)
-      } else if (gasUsed > availableGas) {
+
+      } else if (!enoughGasForDeposit && state.config.exceptionalFailedCodeDeposit) {
```

Comment:
I'm wondering, in this case is it okay that nothing is pushed to the stack? If I understood correctly, on https://github.com/ethereumproject/go-ethereum/blob/master/core/vm/instructions.go#L430 they push a `UInt256.Zero` to the stack